plugins/community/mercari-grpc-federation/v0.9.2/buf.plugin.yaml

```diff
@@ -0,0 +1,10 @@
+version: v1
+name: buf.build/community/mercari-grpc-federation
```

Comment:
cc @goccy

This plugin will be available at buf.build/community/mercari-grpc-federation and can be invoked in any buf.gen.yaml file using `buf generate`.

Reply:
@mfridman Thank you for your support !! 😄